src/components/CourseChecklist/CourseChecklist.test.jsx

```diff
@@ -164,6 +164,20 @@ describe('CourseChecklist', () => {
         expect(listItems).toHaveLength(testChecklistData.length);
       });
 
+      it('the correct number of checks if certificates url is empty', () => {
```

Comment:
The test looks great, but you'll need to change `newProps` once you've made changes to the edx-platform course_validation API that Farhanah recommended.

Reply:
Also, please add a test for weekly highlights, as well, once you add that to the course checklist logic.